readme/the-protocol.md

```diff
@@ -4,57 +4,62 @@ The Monaco Protocol is a liquidity matching program to be deployed on the Solana
 
-The Monaco Protocol offers the ability to wager on binary-outcome event, it is agnostic to the nature of these events. Bet orders can be placed offering to either `BACK` the outcome (it will happen) or to `LAY` it (it won't happen) at a specific price (odds) and stake (wager).
+The Monaco Protocol offers the ability to wager on binary-outcome event, it is agnostic to the nature of these events. Orders can be placed either `FOR` the outcome (it will happen) or to `AGAINST` it (it won't happen) at a specific price (odds) and stake (wager).
 
 ## Markets
 
 - Outcomes
   - Betting markets can be created on anything: from the outcome of a sporting event, the weather in Nantucket on a specific day, to the winner of the next election.
-  - Outcomes on the protocol are represented by an index, in a market of `TEAM_A, DRAW, TEAM_B` a draw result will be a winning index of `1`.
+  - Outcomes on the protocol are represented by an index, in a market with outcomes of `TEAM_A, DRAW, TEAM_B` a draw result will be a winning index of `1`.
 - Price Ladder
   - These markets can have their own price ladder. A sporting event may wish to offer prices ranging from `1.001` to `1000` whilst another market may just offer prices from `1.2` to `5` - these are set by the market operator at time of creation.
 - Token for Entry
   - Markets can be created specifying any [spl-token](https://spl.solana.com/token) as the entry currency, from USDT to an NFT projects native token.
 - Market Lock
-  - Markets are created with a lock time set by a unix timestamp - to the second - in `UTC`. Once this time has elapsed, bet orders can no longer be placed on the market.
+  - Markets are created with a lock time set by a unix timestamp - to the second - in `UTC`. Once this time has elapsed, orders can no longer be placed on the market.
 - Market Title/Type
   - Each market has a human-readable title and a machine-readable type.
 - Event
   - Markets can be linked with an event, this event is an on-chain record of the real-world event the market is associated with. The event will act as the source of truth for outcomes. **Note:** the service supporting events is yet to be released, but markets can still be created and settled independently of their existence. 
 
-## Bet Orders
+## Orders
 
-- Wallets are able to place bet orders one a market.
-- The bet orders contain:
+- Wallets are able to place orders on a market.
+- The order contains:
   - The market the order is to be placed against.
-  - Price of the bet order.
+  - Price of the order.
   - Stake.
   - Index of the predicted winning outcome.
-  - Whether or not the bet order is backing the outcome to happen or laying it not to.
-- Once a bet order is place it is recorded in a market matching pool to allow for matching of orders.
-- Fund are recorded in escrow until a bet order is matched.
-- Bet orders can be cancelled until they are fully matched.
+  - Whether or not the bet order is for the outcome happening or against it.
```

Comment:
I'd maybe do one more search of all files for `bet order`?

Reply:
what the heck!?!?

Reply:
I'm going to go deeper